tembo-pgmq-python/tembo_pgmq_python/queue.py

```diff
@@ -89,6 +89,19 @@ def send(self, queue: str, message: dict, delay: Optional[int] = None) -> int:
             ).fetchall()
         return message[0][0]
 
+    def send_batch(self, queue: str, messages: list[dict], delay: Optional[int] = None) -> list[int]:
+        """Send a batch of messages to a queue"""
+
+        with self.pool.connection() as conn:
+            if delay is not None:
+                # TODO(chuckend): implement send_delay in pgmq
+                raise NotImplementedError("send_delay is not implemented in pgmq")
+            result = conn.execute(
+                "select * from pgmq_send_batch(%s, %s);",
```

Comment:
Latest version of extension has send batch as `pgmq.send_batch()`. And delay is a parameter on that function, default of 0 (no delay).

https://github.com/tembo-io/pgmq/blob/0fad4dc1c03ffa5e7d79cc42e6ec0c6063d24ae7/src/api.rs#L80C24-L81